ftplugin/sml.vim

```diff
@@ -13,24 +13,20 @@ setlocal iskeyword+='
 setlocal commentstring=(*%s*)
 
 " ----- Raimondi/delimitMate -----
-let b:delimitMate_quotes = "\""
+let b:delimitMate_quotes = '"'
 
 " ----- scrooloose/syntastic -----
 " Attempt to detect CM files in SML/NJ checker
-function! s:DetectCM(fname) abort
-  let cm = syntastic#util#findGlobInParent('*.cm', fnamemodify(a:fname, ':p:h'))
-  if cm !=# ''
-    let buf = bufnr(fnameescape(a:fname))
-    call setbufvar(buf, 'syntastic_sml_smlnj_args', '-m ' . syntastic#util#shescape(cm))
-    call setbufvar(buf, 'syntastic_sml_smlnj_fname', '')
-  endif
-endfunction
-
-call s:DetectCM(expand('<amatch>', 1))
```

Comment:
Do you have an intuitive explanation for what the purpose of `<amatch>` was here? When I looked it up, it said that, "When executing autocommands, is replaced with the match which this autocommand was executed." Here, it looks like you've just replaced it with the filename, which is more similar to what it looked like when I implemented it at first.